api/manifest.proto

```diff
@@ -15,6 +15,8 @@
 syntax = "proto3";
 package api;
 
+option go_package = "github.com/tetratelabs/getenvoy-package/api;api";
```

Comment:
@lizan can we move this proto and the things to generate it to tetratelabs/getenvoy? As far as I can tell, the only non-deprecated consumer of this manifest besides getenvoy is https://github.com/tetratelabs/getenvoy-ci/blob/master/go.mod I don't see the sense in remote coupling to a protobuf version over that, as ci should be able to import getenvoy directly if it needed to, right?

Reply:
actually it is used in tetratelabs/getenvoy-ci's manifest/validate_manifest_test.go to validate manifest/manifest.json.. but yeah we may want reconsider where to put these stuff (including manifest.json and this proto)..

Reply:
Yeah we should move this there, no objection from me.

Reply:
I'll own migrating getenvoy-ci following this. thanks for the feedback!